### ISSUE TITLE
get iNat data for US

### ISSUE THREAD

**Issue by coreytcallaghan** [DrJoseValdez/urban_species#3]
Can download from GBIF:

https://www.gbif.org/occurrence/search?country=US&dataset_key=50c9509d-22c7-4a22-a47d-8c48425ef4a7

**Comment by coreytcallaghan** [comment]
Here is a refined filter: https://www.gbif.org/occurrence/download?country=US&dataset_key=50c9509d-22c7-4a22-a47d-8c48425ef4a7&year=2015,2020&coordinateUncertaintyInMeters=0,1000

But just realized that we need to think about threatened species as I'm pretty sure their coordinates are buffered by iNat and/or GBIF. So our spatial matching might not be too accurate. Need to think about the details more here about how they do this...

**Comment by coreytcallaghan** [comment]
Looks like it might not be possible, but I find that hard to believe: https://www.inaturalist.org/pages/help#accesslocation

**Comment by DrJoseValdez** [comment]
Yeah I noticed that. The only thing I can think of is how big is the buffer? If the buffer is a km or less it may be fine if it falls within a city. We can have a rule that if the entire or most of the buffer overlaps a city we can include it. Sure its another step but it may be worth it. Also, perhaps we can find location data from ebird or another site that is much more localized. Most may still have a buffer but perhaps it is smaller. maybe we may need to reduce our taxonomic groups to those with better data. Either way, I had a look and its only a hundred or so total species of birds, amphibians, reptiles, and mammals endangered in the US. Last suggestion is find somewhere that has AOO data for particular groups we can use, then we don't really need iNaturalist data, especially if their data came from iNaturalist. Just a few passing thoughts.

**Comment by DrJoseValdez** [comment]
Since you are in with the iNaturalist and other citizen science groups, do you think you can ask them for the actual datapoints? They state " trusted users, and trusted project curators. iNaturalist Network organizations, in their respective countries, can view true coordinates for taxa set to automatically obscure" This lower the amount of data we have, but better than not having actual location data.

**Comment by coreytcallaghan** [comment]
I have an idea! Let's catch up next time you are in the office.

**Comment by DrJoseValdez** [comment]
I'll be there the rest of the week, stop by if you're around.
